miner/worker.go

```diff
@@ -423,6 +453,18 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 			log.Trace("Skipping transaction with low nonce", "hash", ltx.Hash, "sender", from, "nonce", tx.Nonce())
 			txs.Shift()
 
+		case errors.Is(err, errTxConditionalInvalid):
+			// err contains contextual info on the failed conditional
+			txConditionalRejectedCounter.Inc(1)
+			log.Debug("Skipping account, transaction with failed conditional", "sender", from, "hash", ltx.Hash, "err", err)
```

Comment:
could set atomic rejected to true here rather than having it twice upstream of here

Reply:
but also have the context of the conditional upstream so there are tradeoffs, not super opinionated